Codeinwp/woocommerce-product-addon
# Dashboard changelog

### Summary

Changelog screen added (show PPOM Free and PPOM Pro(if activated) changelogs)
![Screen Shot 2022-11-21 at 12 38 14](https://user-images.githubusercontent.com/25361626/203017121-bd246682-7499-4063-b012-910a877f38d4.png)

### Will affect visual aspect of the product

YES

### Screenshots


### Test instructions

- Visit WP Admin->Left Menu->WooCommerce->PPOM Fields
- Click changelog in header nav
- Make sure the PRO changelog works well if PPOM Pro is installed. (If PPOM Pro is not installed, the Free/Pro dropdown is not shown.)
- Make sure no regression roughly in dashboards.


Closes https://github.com/Codeinwp/woocommerce-product-addon/issues/66.
